assignments/arrays.js

```diff
@@ -63,34 +63,61 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+for (let car of inventory) {
+  if (car.id === 33) {
+    console.log(`Car 33 is a ${car.car_year} ${car.car_make} ${car.car_model}`);
+  }
+}
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(`${inventory[inventory.length - 1].car_make} ${inventory[inventory.length - 1].car_model}`);
```

Comment:
:+1:

Reply:
Might consider assigning the `inventory[inventory.length - 1]` out to a variable to shorten up this line a bit if you need to in the future but great job!